src/main/scala/org/clulab/habitus/actions/HabitusActions.scala

```diff
@@ -170,5 +170,45 @@ class HabitusActions extends Actions {
       )
     }
   }
+  def splitIntoBinary(mentions: Seq[Mention]): Seq[Mention] = {
+    val toReturn = new ArrayBuffer[Mention]()
+    val (target, other) = mentions.partition( m => {
+      val valueLabels = m.arguments("value").map(_.label)
+      valueLabels.length > 1 &&
+      valueLabels.distinct.length == 1
+    }
+    )
+    for (m <- target) {
+      for (v <- m.arguments("value")) {
+        val newArgs = Map("variable" -> m.arguments("variable"), "value" -> Seq(v))
+        val newMention = copyWithArgs(m, newArgs = newArgs)
+        toReturn.append(newMention)
+      }
+    }
+    toReturn ++ other
+  }
+
+  def listStringOverlap(string: String, list: Seq[String]): Boolean = {
+    list.contains(string)
+  }
+
+  def measurementIsAppropriate(m: Mention): Boolean = {
+    // check if any of the possible units shows up in the norm
+    val probablyUnit = m.norms.get.head.split("\\s").last // e.g., get "m2" from "6 m2"; assume value is separated from unit with a space and unit is one token
+    val appropriateUnits = m.label match {
+      case "Quantity" => Seq("t/ha", "kg/ha", "kg", "d", "cm") // fixme: added d and cm for now to make sure tests pass, but do we want days and cm to be quantities?
```

Comment:
This Seq and the above listStringOverlap make me wonder if Set is the way to go because it has an apply method returning a Boolean for membership as if it was intended for this.  For example,
```scala
val quantityUnits = Set("t/ha", "kg/ha", "kg", "d", "cm")
val isAppropriate = quantityUnits(probablyUnit)
```
One could go overboard with something like
```scala
// This can be static in the companion object.  
val labelToAppropriateUnits = Map(
  "Quantity" -> Set("t/ha", "kg/ha", "kg", "d", "cm"),
  "AreaSize" -> Set("ha")
)
// In the method after probableUnit is extracted...
labelToAppropriateUnits
    .getOrElse(m.label, throw new RuntimeException(s"Unknown measurement label ${m.label}"))(probableUnit)
```

Reply:
This looks good, but I am not sure I understand where the labelToAppropriateUnits map should go.

Reply:
@kwalcock I have changed things like you suggested except for the placement of the label/unit map.